pkg/network/node/node.go

```diff
@@ -137,13 +135,11 @@ func New(c *OsdnNodeConfig) (*OsdnNode, error) {
 	case networkutils.NetworkPolicyPluginName:
 		policy = NewNetworkPolicyPlugin()
 		pluginId = 2
-		minOvsVersion = "2.6.0"
-		useConnTrack = true
 	default:
 		return nil, fmt.Errorf("Unknown plugin name %q", networkInfo.PluginName)
 	}
 
-	if useConnTrack && c.ProxyMode == kubeproxyconfig.ProxyModeUserspace {
+	if c.ProxyMode == kubeproxyconfig.ProxyModeUserspace {
 		return nil, fmt.Errorf("%q plugin is not compatible with proxy-mode %q", networkInfo.PluginName, c.ProxyMode)
```

Comment:
Argh. I had forgotten about this part. We still, in theory, allow configuring the userspace proxy if you are using multitenant mode.

_Maybe_ no one is using that in 4.x but I'm not sure we have any way to know.

Sigh. Sorry, I think the simplest thing is to just make it so that ovs-multitenant uses conntrack mode when proxymode is iptables, and non-conntrack mode when proxymode is userspace. So then you can't actually get rid of any of the non-conntrack code and this reverts back to mostly how it was in your first version.

Reply:
I didn't know that either, I'm will close this PR and open a new one on Monday so that we have this code as a reference in the future if we decide